tasks/watchtower.yml

```diff
@@ -0,0 +1,57 @@
+---

```

Comment:
I don't think we need three different variations of one container. Just one with email support is fine, and if the options aren't supplied the emails will simply go nowhere. I think. Thoughts?

Reply:
This is one I struggled with. I wanted the flexibility to support all the notification methods supported (including none) in just one task, but couldn't figure out how to make it work.

I'd be afraid that if email details aren't supplied, it would just generate error spam in the container, which would be problematic if a logging system were implemented (ELK stack or the like).